ecosystem/sep-0006.md

```diff
@@ -1606,6 +1610,16 @@ body should contain error details. For example:
 }
 ```
 
+### Extra object schema
+
+This object is used to pass any additional info to the anchor. Anchors are free to add any field to this object. Anchors
+should notify wallets about fields they are expecting to receive. However, it's recommended to submit a change to this
```

Comment:
How will Anchors communicate this to the wallet? Should the schema be included in the `/info` response?

Reply:
The idea was that this field should be rarely used, and if it is used, anchor should contact wallet directly or print comprehensive error message. 
However, adding schema to `/info` sounds like a good idea. We can only allow simple objects as part of `extra` to simplify definition of the schema